scipy/spatial/_slerp.pyx

```diff
@@ -0,0 +1,29 @@
+import numpy as np
```

Comment:
Honestly, I'm not a fan of preliminarily optimization and "pre-over-engineering", but if you have a plan to use this code from Cython, then let this be.